components/builder-web/bin/start

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 set -x
 
+npm run build-css -- --source-map=true
```

Comment:
What is the reasoning for source-map=true here?  Since we are only doing compression in production, do we need source mapping at all now?

Reply:
This script is only run in dev, and yeah, we do use it in dev.

Reply:
Ok cool